clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -107,6 +118,11 @@ bool CheckedRegionFinder::VisitCompoundStmt(CompoundStmt *S) {
   return false;
 }
 
+bool CheckedRegionFinder::VisitStmtExpr(StmtExpr *SE) {
+  Nwild++;
```

Comment:
Can you explain what the `Nwild` variable is tracking? If it's just saying "there is a wild thing inside the current block" then it could be a boolean, no?

Reply:
You're completely right. Sloppy on my part. I'll fix that up real quick